CIFAR10/train/basenet.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": "v1.0",
+    "author": "bkj",
+    "authorEmail": "ben@canfield.io",
+    "framework": "pytorch",
+    "codeURL": "https://github.com/bkj/basenet/tree/49b2b61e5b9420815c64227c5a10233267c1fb14/examples",
```

Comment:
Please add the command line or instructions on how to reproduce your experiment to a README for this directory or add a `commandLine` field under `misc`